examples/examples_nodejs_test.go

```diff
@@ -24,6 +25,7 @@ func TestAccDedicatedHosts(t *testing.T) {
 
 	integration.ProgramTest(t, &test)
 }
+*/
```

Comment:
Is this meant to be excluded? If yes, could we add a note for why?

Reply:
done!